File: Programmers/Level1/핸드폰 번호 가리기/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        String a = phone_number.substring(phone_number.length()-4, phone_number.length());
        String b = "";
        for(int i=0; i<phone_number.length()-4; i++){
            b+="*";
        }
        answer = b+a;
        return answer;
    }
}

File: Programmers/Level1/핸드폰 번호 가리기/핸드폰 번호 가리기.go
import ("fmt")
func solution(phone_number string) string {
    number:= []rune(phone_number)
    fmt.Print(number)
    length:= len(phone_number)
    
    for i:=0; i<length-4; i++{
        number[i] = '*'
    }
    return string(number)
}
